data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/RecordDispatcherImpl.java

```diff
@@ -106,22 +107,22 @@ public class RecordDispatcherImpl implements RecordDispatcher {
    * @param consumerTracer           consumer tracer
    */
   public RecordDispatcherImpl(
-    final ResourceContext resourceContext,
+    final ConsumerVerticleContext consumerVerticleContext,
     final Filter filter,
     final CloudEventSender subscriberSender,
     final CloudEventSender deadLetterSinkSender,
     final ResponseHandler responseHandler,
     final RecordDispatcherListener recordDispatcherListener,
     final ConsumerTracer consumerTracer,
     final MeterRegistry meterRegistry) {
-    Objects.requireNonNull(resourceContext, "provide resourceContext");
+    Objects.requireNonNull(consumerVerticleContext, "provide resourceContext");
```

Comment:
provide resourceContext: isn't this more consumer context?

Reply:
Yes